Incorrect selection of package version

I have an `eslint-config-myconfig` package in a private registry having the following peer dependencies:

```json
  "peerDependencies": {
    "@typescript-eslint/eslint-plugin": ">=2.21 <3",
    "@typescript-eslint/parser": ">=2.21 <3",
    "eslint": ">=6.8 <7",
    "eslint-config-airbnb-typescript": ">=7 <8",
    "eslint-config-prettier": ">=6.10 <7",
    "eslint-plugin-import": ">=2.20 <3",
    "eslint-plugin-prettier": ">=3.1 <4",
    "eslint-plugin-security": ">=1.4 <2",
    "prettier": ">=1.19 <2",
    "typescript": ">=3.8 <4"
  },
```

When I run `install-peerdeps`, I'm getting a "no matching version" error:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for typescript@3.8.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

It appears that install-peerdeps is not using the exact same semver expression when issuing the "npm install" command. So, instead of **`npm install typescript@">=3.8 <4"`**, it's just reading the first literal a.b.c number.

A quick debug shows the command being executed:

```js
npm [
  'install',
  'eslint-config-insight@0.1.0',
  '@typescript-eslint/eslint-plugin@2.21.0',
  '@typescript-eslint/parser@2.21.0',
  'eslint@6.8.0',
  'eslint-config-airbnb-typescript@7.0.0',
  'eslint-config-prettier@6.10.0',
  'eslint-plugin-import@2.20.0',
  'eslint-plugin-prettier@3.1.0',
  'eslint-plugin-security@1.4.0',
  'prettier@1.19.0',
  'typescript@3.8.0',
  '--save-dev'
]
```

--- a/src/install-peerdeps.js
+++ b/src/install-peerdeps.js
@@ -247,7 +247,8 @@ function installPeerDeps(
           // Remove -0
           return packageName.substr(0, packageName.length - 2);
         }
-        return packageName;
+        // Fix #64 -- add quotes
+        return `"${packageName}"`;
       }
       // If we have spaces in our args spawn()
       // cries foul so we'll split the packagesString

--- a/src/install-peerdeps.js
+++ b/src/install-peerdeps.js
@@ -154,12 +154,9 @@ const getPackageString = ({ name, version }) => {
   if (version.indexOf(" ") >= 0) {
     // Semver ranges can have a join of comparator sets
     // e.g. '^3.0.2 || ^4.0.0' or '>=1.2.7 <1.3.0'
-    // Take each version in the range and find the maxSatisfying
-    const rangeSplit = version
-      .split(" ")
-      .map(v => coerce(v))
-      .filter(v => valid(v));
-    const versionToInstall = maxSatisfying(rangeSplit, version);
+    // Take the last version in the range
+    const rangeSplit = version.split(" ");
+    const versionToInstall = rangeSplit[rangeSplit.length - 1];
 
     if (versionToInstall === null) {
       return name;

